### PR TITLE
Portfolio Mobile skeleton

### PR DESCRIPTION
**Pull request by telanet**
### In this milestone I:

- Set up a new repository using best practices
- Parse a Figma design to create a UI
- Build the first two sections of the mobile website(toolbar and headline)
- Use flexbox on the 2 sections

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+.intro {
+  display: flex;
+  flex-direction: column;
+  padding: 0 24px 0 24px;
+  background-color: #1c1a19;
+  justify-content: center;
+  height: 590px;
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  background-image: url(/img/Illustration.png);
```
Comment (sja-thedude) [inline style.css:55]: - [ ] Make sure this background image works. Right now it's not displaying the background image in your design. Kindly fix this.
- Your design -
Screenshot 2022-03-15 at 8 52 35 PM

- Figma design -
Screenshot 2022-03-15 at 8 52 11 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @telanet, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 👏 

## Highlights
- Descriptive Readme 👍 
- All linters are passing 👍 
- Descriptive PR title and summary 👍 

- [ ] Kindly _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.intro h1 {
+  margin: 0;
+  font-size: 56px;
+  color: #ff6b00;
```
[the remark this answers is not in the document]
Comment (Kingstalux) [reply style.css:62]: resolved 👍

File: style.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.intro h3 {
+  color: #fff5e1;
+  margin: 0;
+}
```
[the remark this answers is not in the document]
Comment (Kingstalux) [reply style.css:70]: resolved 👍

File: style.css (hunk 1 above)
Comment (Kingstalux) [reply style.css]: not resolved

**Review by Kingstalux — APPROVED** [review]
Hi @ksigei,
# Status: Approved ✔️ ✔️ ✔️ 
 
128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**